data/sql/updates/pending_db_world/rev_1629752061588917431.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629752061588917431');
+
+UPDATE `creature` SET `MovementType` = 1, `wander_distance` = 3  WHERE (`guid` IN (50173, 50175, 50176, 50177, 50179, 50180, 50194, 50195, 50196, 50198, 50203, 50205, 50206, 50215, 50216, 50217, 50218, 50222, 50227, 50230, 50235, 50244, 50250, 50251, 50252, 50253, 50254, 50291, 50297, 50307, 50313, 50314, 50317, 50319));
```

Comment:
It looks like creatures also have a MovementType in their templates - did you consider updating the movement type in `creature_template` as well?

https://www.azerothcore.org/wiki/creature#movementtype
https://www.azerothcore.org/wiki/creature_template#creature_template-MovementType